api/src/main/java/io/strimzi/api/kafka/model/status/ListenerStatus.java

```diff
@@ -49,6 +50,16 @@ public void setType(String type) {
         this.type = type;
     }
 
+    @Description("The name of the listener. " +
+            "Can be one of the following three types: `plain`, `tls`, and `external`.")
```

Comment:
I would avoid saying this here. Maybe it's better saying that for now it's reflecting the same value as type but when it will be deprecated, this field will report the actual name of the listener. @tombentley @scholzj wdyt?

Reply:
This is obviously wrong even for the `type` field and should be fixed.

Reply:
Right this is still describing the older meaning of the `type` field.

Reply:
Do I need to change the description of the `name` field only or do I need to change description of `type` field also?

Reply:
@scholzj @ppatierno

Reply:
You should change both. They mean the same.

Reply:
@scholzj @ppatierno does the description for both now look fine?